pkg/placement/raft/server.go

```diff
@@ -172,7 +191,7 @@ func (s *Server) StartRaft(config *raft.Config) error {
 		return err
 	}
 
-	logging.Debug("Raft server is starting")
+	logging.Infof("Raft server is starting on %s...", s.raftBind)
```

Comment:
```suggestion
	logging.Infof("Raft server is starting on %q...", s.raftBind)
```

Reply:
s.raftBind is string. why do we need %q ?

Reply:
quote that particular string?

Reply:
I do not think it needs double quote :)